SeleniumPractice/src/test/java/SeleniumDevTest.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.util.ArrayList;
+
+public class SeleniumDevTest {
+
+    @Test
+    public void lastVersion_3_141_59() throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+
+        driver.manage().window().maximize();
+        driver.get("https://www.selenium.dev/");
+        Thread.sleep(1000);
+
+        WebElement download = driver.findElement(By.xpath("//*[@id=\"navbar\"]/a[1]"));
+        download.click();
+
+
+        WebElement latestStableVersion = driver.findElement(By.xpath("//p[contains(text(), 'Latest stable version')]/child::a[contains(text(), '3.141.59')]"));
```

Comment:
Not a good idea, because even if it fails, you will not be sure what was the reason. Just find the latest version and check if it's your expected value or not via assertion.